ui/components/ui/download-threatscore-button/download-threascore-button.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import { Tooltip } from "@nextui-org/react";
+import { DownloadIcon } from "lucide-react";
+
+import { CustomButton } from "../custom/custom-button";
+
+interface DownloadThreatscoreButtonProps {
+  paramId: string;
+  onDownload: (paramId: string) => void;
+  ariaLabel?: string;
+  isDisabled?: boolean;
+  textTooltip?: string;
+  isDownloading?: boolean;
+}
+
+export const DownloadThreatscoreButton = ({
```

Comment:
there’s another component called `ThreatscoreDownloadButton` what’s the difference? do we need both?